### DISCUSSION TITLE
本地开发正常，但是在preview的时候页面空白

### DISCUSSION THREAD

**Discussion by qgq972266356** [alex8088/electron-vite#553] [Q&A]
<img width="760" alt="image" src="https://github.com/alex8088/electron-vite/assets/38481156/048a5f85-75f1-4caf-b26e-801f6352be4b">

**Comment by alex8088** [comment]
https://cn.electron-vite.org/guide/troubleshooting#vue-router-%E6%88%96-react-router-dom-%E5%BC%80%E5%8F%91%E6%A8%A1%E5%BC%8F%E6%AD%A3%E5%B8%B8-%E4%BD%86%E5%9C%A8%E6%9E%84%E5%BB%BA%E5%90%8E%E6%97%A0%E6%95%88
